Demo/Demo/Models/Order.swift

```diff
@@ -22,6 +23,13 @@ struct Order: Codable, Equatable {
         let attributes: Attributes?
     }
 
+    struct PayPal: Codable, Equatable {
+
+        let emailAddress: String
+        let accountId: String
```

Comment:
```suggestion
        let accountID: String
```

Reply:
To use codingKey to change this, I have to include every field from the response in the CodingKeys, I think?
```
"payment_source": {
        "paypal": {
            "email_address": "me@personal.example.com",
            "account_id": "someID",
            "account_status": "VERIFIED",
            "name": {
                "given_name": "John",
                "surname": "Doe"
            },
            "address": {
                "country_code": "US"
            }
        }
    }
```